components/artist-projects.js

```diff
@@ -0,0 +1,25 @@
+import ArtistProjectPreview from '../components/artist-project-preview'
+
+export default function ArtistProjects({ projects }) {
```

Comment:
Note to us: this component seems to be extremely similar to more-stories.js (haha, needs to be renamed)

Lets keep an eye on these and figure out how to reduce duplication after launch!